app/jobs/create_gemini_recipient_ids_job.rb

```diff
@@ -0,0 +1,31 @@
+# Creates the Uphold Cards for a publisher
+class CreateGeminiRecipientIdsJob < ApplicationJob
+  queue_as :default
+
+  def perform(gemini_connection_id:)
+    gemini_connection = GeminiConnection.find(gemini_connection_id)
+    # Users aren't able to create a recipient id if they are not fully verified
+    if gemini_connection.payable?
+      recipient = Gemini::RecipientId.find_or_create(token: gemini_connection.access_token)
+      gemini_connection.update(recipient_id: recipient.recipient_id)
+      gemini_connection.update_default_currency
```

Comment:
```suggestion
```

If i recall correctly, there is no default currency for Gemini. @Miyayes would you confirm?

Reply:
Confirmed we can remove it.